arp_scripts/submit_small_data.sh

```diff
@@ -90,8 +90,8 @@ LCLS2_HUTCHES="rix, tmo"
 if echo $LCLS2_HUTCHES | grep -iw $HUTCH > /dev/null; then
     echo "This is a LCLS-II experiment"
     source /cds/sw/ds/ana/conda2/manage/bin/psconda.sh
-    conda deactivate
-    conda activate ps-4.2.6
+    #conda deactivate
```

Comment:
Any reason to leave this commented?

Reply:
Mostly only to quickly see where this would go should we need to do this. having cleaner code is probably more important though.